src/components/AppLoading.tsx

```diff
@@ -47,14 +32,14 @@ const LoadingSubText = styled.h6`
   text-align: center;
 `;
 
-export const AppLoading = () => {
-  return (
-    <Layout>
-      <DashboardPanelLoading>
-        <Loader className="loading" />
-        <LoadingText>{translateRaw('APP_LOADING')}</LoadingText>
-        <LoadingSubText>{translateRaw('APP_LOADING_SUBHEADER')}</LoadingSubText>
-      </DashboardPanelLoading>
-    </Layout>
-  );
-};
+const AppLoading = () => (
```

Comment:
This component now looks wrong, it has a very low width. It's difficult to take a screenshot of, but is clear if you go to a page that you haven't already loaded.

Reply:
Solved with `width: 100%`